package.json

```diff
@@ -162,7 +161,9 @@
     "vite-plugin-top-level-await": "^1.6.0"
   },
   "optionalDependencies": {
-    "electron": "^38.2.1"
+    "electron": "^38.2.1",
+    "@lydell/node-pty": "1.1.0",
+    "node-pty": "1.1.0-beta39"
```

Comment:
This should not be needed at all anymore.

Reply:
i need regular node-pty for electron